require/require.go

```diff
@@ -33,6 +40,20 @@ func Contains(t TestingT, s interface{}, contains interface{}, msgAndArgs ...int
 	}
 }
 
+// Containsf asserts that the specified string, list(array, slice...) or map contains the
+// specified substring or element.
+//
+//    assert.Containsf(t, "Hello World", "World", "extra error message")
```

Comment:
The examples for `Containsf` should contain a formatting string + arguments. Otherwise `Contains` is fine.